model_signing/signature/encoding.py

```diff
@@ -16,6 +16,9 @@
 
 PAYLOAD_TYPE = "application/vnd.in-toto+json"
 
+# The proto contains a Statement message.
+# pylint: disable=no-member
+
```

Comment:
Can we disable this locally and re-enable as soon as we no longer need it disabled?